src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length - 1);
+        return Colors.values()[index].name();
```

Comment:
```suggestion
public class ColorSupplier {
    private static final Random random = new Random();
        
    public String getRandomColor() {
        int index = random.nextInt(Colors.values().length - 1);
        return Colors.values()[index].name();
```
better make objects like this global